tempodb/encoding/vparquet3/combiner.go

```diff
@@ -86,6 +86,9 @@ func (c *Combiner) ConsumeWithFinal(tr *Trace, final bool) (spanCount int) {
 		c.result.RootSpanName = tr.RootSpanName
 	}
 	c.result.DurationNano = c.result.EndTimeUnixNano - c.result.StartTimeUnixNano
+	if len(c.result.ServicesMeta) == 0 {
```

Comment:
shouldn't this make an effort to combine? instead of just choosing one side or another. i think the only correct way to do it would be to just do the calculation after the combine is complete in the `Result()` function.

Reply:
Yep, probably using the same algorithm as in `pkg/traceql/combine.go` (currently empty). I think I could just combine two `[]ServicesMeta` by the service name and sum up their spanCount and errorCount?

In this case it'd also beneficial to store it as a `map[string]ServicesMeta` (keyed by service name) instead of `[]ServicesMeta`, but I'm not sure how good that'll work with parquet, or if keeping it as a slice is better for interoperability.

Reply:
I've updated the PR to accumulate the spanCount and errorCount now.